Meadowlark-js/packages/meadowlark/test/http/remote.deployment.test.http

```diff
@@ -0,0 +1,255 @@
+# This file is meant to be used in Visual Studio Code with the popular "REST Client" extension.
+
+# This file will allow you to quickly exercise many of the basic features of the Meadowlark API,
+# Each HTTP request is part of a larger scenario, and is meant to be sent in order. The comment
+# above each HTTP action explains the request and expected result (sometimes an error). Click
+# on "Send Request" in between the comment and the HTTP action to execute the request.
+
+# Note that delays of a few seconds on the first call to an HTTP action are due to Lambda cold starts.
+# For this proof-of-concept, the MetaEd model is being loaded and Meadowlark's API surface
+# is being computed and cached on the Lambda. Subsequent calls to the same HTTP action (e.g. POST)
+# will be much faster, at least until AWS reclaims that Lambda due to inactivity.
+
+# Ensure you have followed the DEPLOYMENT.md instructions. You should have deployed with ACCESS_TOKEN_REQUIRED=true
+# and have loaded descriptors with "yarn load:descriptors:aws-dev".
+
+# **** Change this to be the hostname portion of the endpoint URLs displayed at the end of the deployment process.
+@hostname=abcdefg.execute-api.us-east-1.amazonaws.com
+
+
+# FYI, this is a non-expiring bearer token that is valid in the default configuration for the OAuth 2.0 proof-of-concept. 
+@token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJlZC1maS1tZWFkb3dsYXJrIiwiYXVkIjoibWVhZG93bGFyayIsInN1YiI6InN1cGVyLWdyZWF0LVNJUyIsImp0aSI6IjNkNTliNzVmLWE3NjItNGJhYS05MTE2LTE5YzgyZmRmOGRlMyIsImlhdCI6MTYzNjU2MjA2MCwiZXhwIjozODQ1NTQ4ODgxfQ.WF5ABFZAvNsLDZktwa8VxDR846fGptRXJObtQitbL8I
```

Comment:
I'm going to make three changes here:

1. Adding `@stage1` token
2. Adding a call to the base API endpoint to "verify service is running"
3. Changing from `http` to `https`.